README.md

```diff
@@ -10,9 +10,33 @@
 Small package to visualize your [Ferrite.jl](https://github.com/Ferrite-FEM/Ferrite.jl) results. Currently supports only Makie,
 but the goal is to extend it for different plotting packages.
 
+The package is highly experimental and breaking changes about the internal machinery are about to come.
+Likely, only a small fraction of the interface will change over time.
+
 ## Installation
 
 ```julia
 pkg> add FerriteVis
 ```
 
+
+## Features
+
+- `solutionplot` FE solution contour plot on arbitrary finite element mesh (in Makie called `mesh` plots)
+- `ferriteviewer` viewer with toggles and menus that update the plot
+- `wireframe` plots the finite element mesh and optionally labels nodes and cells
+- `arrows` - also called `quiver` plots, in paraview `glyph` filter
+- `surface` 2D solutions in 3D space as surface, in paraview `warp by scalar` filter
+- live plotting while your simulation runs with any of the above listed options
```

Comment:
We should note that our visualizer works just synchronous to the simulation (in contrast to viewers like e.g. GLVis and ParaView Catalyst, which work async)